recipe/meta.yaml

```diff
@@ -6,35 +6,43 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/googlemaps-{{ version }}.tar.gz
-  sha256: 3055fcbb1aa262a9159b589b5e6af762b10e80634ae11c59495bd44867e47d88
+  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v4.10.0.tar.gz
+  sha256: cc07f739731ba4e0a4586543ad14470695addff8f30840748b9c3b89e393aa36
 
 build:
-  noarch: python
+  skip: True  # [py<35]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
 
 requirements:
   host:
-    - python >=3.7
-    - requests >=2.20.0,<3.0
+    - python
     - pip
+    - setuptools
+    - wheel
+    - requests >=2.20.0,<3.0
```

Comment:
```suggestion
```
`requests` isn't needed to build the package even if it's mentioned in `setup_requires`, IMHO, mistakenly.

Reply:
You were right, I patched setup.py, and the build runs fine without this dependency.